# define box volume which encapsulate the car (set values in os1_sensor frame)
# see devops wiki page 'LiDAR FoV Trimming' for more information
minX: -300
maxX: 0.0
minY: -300
maxY: 300
minZ: -100
maxZ: 100
input_topic: "os1_cloud_node/points"